res/test.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
You didn't choose any specific testing framework, but how can You measure code coverage now?